src/rules/noUnusedVariableRule.ts

```diff
@@ -65,7 +65,14 @@ export class Rule extends Lint.Rules.AbstractRule {
     };
     /* tslint:enable:object-literal-sort-keys */
 
-    public static FAILURE_STRING_FACTORY = (name: string) => `Unused variable: '${name}'`;
+    public static FAILURE_TYPE_BIND_ELEM = "binding element";
+    public static FAILURE_TYPE_FUNC = "function";
+    public static FAILURE_TYPE_IMPORT = "import";
+    public static FAILURE_TYPE_PARAM = "parameter";
+    public static FAILURE_TYPE_PROP = "property";
+    public static FAILURE_TYPE_VAR = "variable";
```

Comment:
I don't think they can since they are in the `Rule` class, and not the `NoUnusedVariablesWalker` class.  I could move them to the walker class if desired, but I figured that all the failure message related text should stay together.